tests/errors/pure_warnings.py

```diff
@@ -21,9 +21,9 @@ def main():
     foo11: Bar[:, :, :]  # warning
     foo12: cython.int[:, ::1]
     with cython.annotation_typing(False):
-        foo8: Bar = 1
-        foo9: stdint.bar = 5
-        foo10: cython.bar = 1
+        foo13: Bar = 1
+        foo14: stdint.bar = 5
+        foo15: cython.bar = 1
```

Comment:
I think this naming reuse was intentional. It's debatable if it should be an error to do this if annotation typing is disabled. Static type checkers would probably reject it, though, so maybe the tested assumption itself is wrong. But it's something we allowed before, apparently.